drivers/mysql/mysql_test.go

```diff
@@ -95,11 +95,20 @@ func (suite *MysqlTestSuite) TestOpen() {
 	})
 
 	suite.T().Run("when connURL is valid and bare uses default configuration", func(t *testing.T) {
-		connectedDriver, teardown := suite.InitializeDriver(testConnURL)
+		connectedDriver, teardown := suite.InitializeDriver(defaultConnURL)
```

Comment:
Any reason for this to change?

Reply:
Yeah it should be the `defaultConnURL` because we are checking things against default config. And it was modifying the default config before. And it was trying to compare same things.